lib/twirp/rails/logging/subscriber.rb

```diff
@@ -16,24 +16,40 @@ def instrumenter(event)
 
         def default_log_writer(event)
           twirp_call_info = {
-            "service" => event.payload[:env][:service].try(:full_name),
-            "method" => event.payload[:env][:rpc_method],
-            "path" => event.payload[:rack_env]["REQUEST_PATH"],
-            "duration" => event.duration
+            **status_info(event),
+            **base_log_info(event)
           }
 
-          if (twerr = event.payload[:twerr])
-            twirp_call_info["error_code"] = twerr.code
+          twirp_call_info[:params] = event.payload[:env][:input].to_h if Twirp::Rails.configuration.logging.log_params
+
+          if (exception = event.payload[:env][:exception]) && Twirp::Rails.configuration.logging.log_exceptions
+            twirp_call_info[:exception] = exception
           end
 
-          twirp_call_info["params"] = event.payload[:env][:input].to_h if Twirp::Rails.configuration.log_params
+          data = Twirp::Rails.configuration.logging.log_formatter.call(twirp_call_info)
+          logger.info data
+        end
 
-          if (exception = event.payload[:env][:exception]) && Twirp::Rails.configuration.log_exceptions
-            twirp_call_info['exception'] = exception
+        def base_log_info(event)
+          {
+            service: event.payload[:env][:service].try(:full_name),
+            method: event.payload[:env][:rpc_method],
+            path: event.payload[:rack_env]["REQUEST_PATH"],
+            time: Time.current.iso8601,
+            duration: event.duration
+          }
+        end
+
+        def status_info(event)
+          status = {}
+
+          if (twerr = event.payload[:twerr])
```

Comment:
Is `twerr` a typo?

Reply:
Nah just shorthand passed from the event payload.